### ISSUE TITLE
breaking change in @nestjs/bullmq v10.2.0

### ISSUE THREAD

**Issue by MuhAssar** [nestjs/bull#2203]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

```
BullModule.registerQueue({
      name: 'ImportLexicon',
      processors: [{ path: join(__dirname, './isolated/importLexicon.js') }],
    })
```

this code worked perfectly in v10.1.0

but in v10.2.0 it throws the following error:

`Type '{ path: string; }' is not assignable to type 'BullQueueProcessor'.
    Property 'connection' is missing in type '{ path: string; }' but required in type 'BullQueueAdvancedSeparateProcessor'.`

even in nestjs docs it says how to use it exactly as I did https://docs.nestjs.com/techniques/queues#separate-processes-1 :

```
import { Module } from '@nestjs/common';
import { BullModule } from '@nestjs/bull';
import { join } from 'path';

@Module({
  imports: [
    BullModule.registerQueue({
      name: 'audio',
      processors: [join(__dirname, 'processor.js')],
    }),
  ],
})
export class AppModule {}
```

so this is clearly either a bug or a breaking change

### Minimum reproduction code

already mentioned in the body

### Expected behavior

no errors when the processors only contain [{path:string}]

### Package version

10.2.0

### Bull version

5.12.0

### NestJS version

10.1.3

### Node.js version

20.11.0

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by kamilmysliwiec** [comment]
Please provide a [minimum reproduction repository](https://minimum-reproduction.wtf/) (Git repository/[StackBlitz](https://stackblitz.com/fork/github/nestjs/typescript-starter)/[CodeSandbox](https://codesandbox.io/s/github/nestjs/typescript-starter/tree/master) project).

**Comment by MuhAssar** [comment]
minimum reproduction repo:
https://codesandbox.io/p/devbox/inspiring-rhodes-lmynkp?workspaceId=f37e87fd-4744-4fbd-b916-f9cf589b8b39

run:
npm run start

observe the error:
```
src/app.module.ts:11:20 - error TS2322: Type '{ path: string; }' is not assignable to type 'BullQueueProcessor'.
  Property 'connection' is missing in type '{ path: string; }' but required in type 'BullQueueAdvancedSeparateProcessor'.

11       processors: [{ path: join(__dirname, 'processor.js') }],
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/bullmq/dist/esm/interfaces/queue-options.d.ts:15:5
    15     connection: ConnectionOptions;
           ~~~~~~~~~~
    'connection' is declared here.

Found 1 error(s).
```


now open package.json and change :
```
"@nestjs/bullmq": "10.2.0",
```
to
```
"@nestjs/bullmq": "10.1.0",
```

then run:
npm i
npm run start

observer no error

**Comment by kamilmysliwiec** [comment]
Thank you @MuhAssar! Fixed in 10.2.1
